src/views/Alteration.vue

```diff
@@ -240,7 +240,7 @@ export default class Alteration extends Mixins(
     // bypass this when Jest is running as FF are not fetched
     if (!this.isJestRunning && !getFeatureFlag('alteration-ui-enabled')) {
       window.alert('Alterations are not available at the moment. Please check again later.')
-      this.$root.$emit('go-to-dashboard')
+      this.$root.$emit('go-to-dashboard', true)
```

Comment:
Go to dashboard and bypass unsaved changes check.

(Same thing in the other filing views.)